samples/sdl_gamecontroller_event/main.c

```diff
@@ -0,0 +1,57 @@
+#include <xboxrt/debug.h>
+#include <pbkit/pbkit.h>
+#include <hal/xbox.h>
+#include <assert.h>
+#include <SDL.h>
+#include <stdbool.h>
+
+int main() {
+  int ret = pb_init();
+  if (ret != 0) {
+    XSleep(2000);
+    return -1;
+  }
+
+  pb_show_debug_screen();
+
+  SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER);
+
+  int gameControllerIndex = 0;
+  SDL_GameController *gameController = SDL_GameControllerOpen(gameControllerIndex);
+  if (gameController == NULL) {
+    debugPrint("Could not open GameController %i, exiting...", gameControllerIndex);
+    SDL_QuitSubSystem(SDL_INIT_GAMECONTROLLER);
+    XSleep(2000);
+    return -1;
+  }
+
+  debugPrint("Num joysticks      : %02i\n", SDL_NumJoysticks());
```

Comment:
1. Joystick is not interesting for a gamecontroller sample
2. This message suggests that multiple gamecontrollers will react (they don't)

Unless the following code is ran for each gamecontroller, this message should be replaced with one that suggests that the gamecontroller in "Port 1" is being used.